install.sh

```diff
@@ -78,6 +88,7 @@ function install_dependencies()
         if ! [ -x "$(command -v swig)" ]
         then
             install_swig_from_sources
+            install_seed_voicecard_driver
```

Comment:
Why is `install_seed_voicecard_driver` called from within `install_flite_from_source`? 
It should be called from the main thread

Reply:
@woshikie It is not called from `install_flite_from_source`. You are cheated by the "diff" view.